test/specs/parallel.js

```diff
@@ -5,10 +5,10 @@ describe('parallel', () => {
     beforeEach(clean)
 
     it('should run several suites in parallel', () => {
-        return run(['broken', 'failing']).then((results) => {
+        return run(['long-test-one', 'long-test-two']).then((results) => {
```

Comment:
And the last hack. I belive it isn't good test initially. It passes in my PC, but sometimes fails in Travice.